SU-SWS/stanford_image
# SOE-1713: add stanford_fw_banner_tall_caption

# NOT READY FOR REVIEW

# Summary
-  add stanford_fw_banner_tall_caption view mode

# Needed By (Date)

# Urgency

# Steps to Test

1. Add this to an SoE Stanford Page
2. Verify it works

# Affected Projects or Products
- Prerequiste for SoE Helper *redesign* branch (https://github.com/SU-SOE/stanford_soe_helper) and Stanford Magazine (https://github.com/SU-SWS/stanford_magazine)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1713
- Depends on https://github.com/SU-SWS/stanford_image_styles/pull/34

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
